pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: package_rename
 description: A Blazingly Fast way to configure your awesome flutter project to be production ready.
-version: 1.2.0
+version: 1.2.1
```

Comment:
I'll change it when releasing a new publish according to the commits made in the meantime.

Reply:
Resolved in [new commit](https://github.com/obemu/package_rename/commit/23fccf09eeb92780e0bd0e10b45c231503e02128)